mppm/release-notes/release-notes.md

```diff
@@ -4,6 +4,16 @@ title: Release notes
 description: Release notes for Multiplayer Play Mode including new features, updates, bug fixes, and known issues.
 ---
 
+## [1.4.0-pre.1] - 2024-12-05
+
+### Fixed
+- Virtual Player windows correctly update and stay in sync when you enter and exit Play mode.
```

Comment:
I assumed here that  "clone editor VP Windows" means "Virtual Player windows" but correct me if I'm wrong.

Reply:
Yes exactly right :)